### ISSUE TITLE
Styles that only do exclude_* should include the default set by default

### ISSUE THREAD

**Issue by jaymzh** [markdownlint/markdownlint#369]
When we parse -r, we are very careful to handle cases where only exclude rules are passed in (all rules, minus those), but we don't do the same thing for styles.

So if, in a custom style, you do:

```ruby
exclude_rule 'MD004'
```

You will get *no* rules. That's really not user friendly. That should be the default set, minus MD004. To get that, you must do:

```
all
exclude_rule 'MD004'
```

Which is OK, but non-intuitive.
